com.unity.render-pipelines.universal/Runtime/UniversalRenderPipelineGlobalSettings.cs

```diff
@@ -0,0 +1,240 @@
+                    return null;
+                }
+            }
+            Debug.Assert(assetCreated, "Could not create URP's Global Settings - URP may not work correctly - Open  Project Settings > Graphics > URP Settings for additional help.");
+            UpdateGraphicsSettings(assetCreated);
+            return UniversalRenderPipelineGlobalSettings.instance;
+        }
+
+        internal static UniversalRenderPipelineGlobalSettings Create(string path, UniversalRenderPipelineGlobalSettings src = null)
+        {
+            UniversalRenderPipelineGlobalSettings assetCreated = null;
+
+            // make sure the asset does not already exists
+            assetCreated = AssetDatabase.LoadAssetAtPath<UniversalRenderPipelineGlobalSettings>(path);
+            if (assetCreated == null)
+            {
+                assetCreated = ScriptableObject.CreateInstance<UniversalRenderPipelineGlobalSettings>();
+                AssetDatabase.CreateAsset(assetCreated, path);
+                if (assetCreated != null)
+                {
+                    assetCreated.name = System.IO.Path.GetFileName(path);
+                }
+            }
+
+            if (assetCreated)
+            {
+                if (src != null)
+                {
+                    assetCreated.lightLayerName0 = System.String.Copy(src.lightLayerName0);
+                    assetCreated.lightLayerName1 = System.String.Copy(src.lightLayerName1);
+                    assetCreated.lightLayerName2 = System.String.Copy(src.lightLayerName2);
+                    assetCreated.lightLayerName3 = System.String.Copy(src.lightLayerName3);
+                    assetCreated.lightLayerName4 = System.String.Copy(src.lightLayerName4);
+                    assetCreated.lightLayerName5 = System.String.Copy(src.lightLayerName5);
+                    assetCreated.lightLayerName6 = System.String.Copy(src.lightLayerName6);
+                    assetCreated.lightLayerName7 = System.String.Copy(src.lightLayerName7);
+                }
+                else
+                {
+                }
+
+                AssetDatabase.SaveAssets();
+                AssetDatabase.Refresh();
+            }
+
+            return assetCreated;
+        }
+
+#endif
+
+        void Reset()
+        {
+            ResetLightLayerNames();
+            UpdateRenderingLayerNames();
+        }
+
+        [System.NonSerialized]
+        string[] m_RenderingLayerNames;
+        string[] renderingLayerNames
+        {
+            get
+            {
+                if (m_RenderingLayerNames == null)
+                {
+                    UpdateRenderingLayerNames();
+                }
+
+                return m_RenderingLayerNames;
+            }
+        }
+        public string[] renderingLayerMaskNames => renderingLayerNames;
+
+        void UpdateRenderingLayerNames()
+        {
+            if (m_RenderingLayerNames == null)
+                m_RenderingLayerNames = new string[32];
```

Comment:
Initially they were 24 for URP but that was inconsistent with HDRP and Default convention. Let me know if I should revert back to 24.

Reply:
@kaychang-unity we agreed to change it to 32 for URP.